doc/nrf/samples/wifi_zephyr.rst

```diff
@@ -0,0 +1,48 @@
+.. _wifi_zephyr_samples:
+
+Wi-Fi: Zephyr networking samples
+################################
+
+In addition to nRF Connect SDK samples, it is also possible to run selected Zephyr networking
+samples with Wi-Fi.
+The following list specifies samples that are currently supported with the nRF700x driver:
+
+* :ref:`dhcpv4-client-sample`
+* :ref:`dns-resolve-sample`
+* :ref:`ipv4-autoconf-sample`
+* :ref:`mdns-responder-sample`
+* :ref:`mqtt-publisher-sample`
+* :ref:`mqtt-sn-publisher-sample`
+* :ref:`coap-client-sample`
+* :ref:`coap-server-sample`
+* :ref:`sockets-echo-sample`
+* :ref:`async-sockets-echo-sample`
+* :ref:`sockets-echo-client-sample`
+* :ref:`sockets-echo-server-sample`
+* :ref:`sockets-http-get`
+* :ref:`sntp-client-sample`
+* :ref:`syslog-net-sample`
+* :ref:`telnet-console-sample`
+
+Building and running
+********************
+
+All of the listed samples have been provided with an overlay file, ``overlay-nrf700x.conf``, which
+allows to configure the sample to run with nRF700x driver.
```

Comment:
Is "nRF700x driver" the right name? IIRC, we use "Wi-Fi driver" in NCS, but up to finding a better name if needed.